src/Client.php

```diff
@@ -190,18 +196,18 @@ public function prepareOAuth()
                     $this->oauthConfig,
                     $this->getSignableParameters()
                 );
-                $requestHeaders = $this->getRequest()->getHeaders();
+                $requestHeaders   = $this->getRequest()->getHeaders();
                 $requestHeaders->addHeaders(['Authorization' => $oauthHeaderValue]);
                 break;
             case OAuth::REQUEST_SCHEME_POSTBODY:
-                if ($this->getRequestMethod() == HttpRequest::METHOD_GET) {
+                if ($this->oauthConfig->getRequestMethod() === HttpRequest::METHOD_GET) {
```

Comment:
I mean, why change `this->getRequestMethod()` to `this->oauthConfig->getRequestMethod()` ? Is this a bug fix? Then separate PR is needed

Reply:
Not a bug fix: instead of `Client::__call('getRequestMethod')`, `Client::$oathConfig->getRequestMethod()`

Reply:
Just keep original code, and replace `==` with `===` for cs, as compare with string is mostly safe,

 bug fix if any, should be in separate PR to ease revert in case of regression

Reply:
@gsteel if you can verify, then ok 👍

Reply:
I'm with you on the equality operator @samsonasik.

@manishranjan-adobe Please revert to `==`:

```suggestion
                if ($this->oauthConfig->getRequestMethod() == HttpRequest::METHOD_GET) {
```

Reply:
You may need to add an inline phpcs suppression along the lines of `/** phpcs:ignore Some.Rule.Name */`

Reply:
@gsteel using `===` is fine :), as it compare to pure string, not numeric string, `getRequestMethod()` mostly won't return string-able object

Reply:
I'm thinking more about case-sensitivity

Reply:
`==` is unrelated with case sensitivity https://3v4l.org/7QMtP